src/universal/components/Avatar/Avatar.js

```diff
@@ -164,8 +163,11 @@ s = StyleSheet.create({
   avatarLargest: {
     fontSize: theme.typography.s6
   },
-
+  // TODO terry help me make this not look ugly
```

Comment:
@ackernaut 1 more todo item for ya. i gave everyone a white border so the avatars always take up the same amount of space so there's no slight movements when changing facilitators.